src/App.js

```diff
@@ -3,19 +3,17 @@ import React from 'react';
 import './App.css';
 function MyList(props) {
   const items = props.items;
-  const listItems = numbers.map((item) =>
+  const listItems = items.map((item) =>
     <li>{item}</li>
   );
   function App() {
     return (
       <ul>{listItems}</ul>
     );
   }
```

Comment:
Can you explain what this `App` doing? what is the need for this?

Reply:
As I titled the pull request

Reply:
I want more detail